src/pages/gen2/build-a-backend/data/data-modeling/secondary-index/index.mdx

```diff
@@ -125,15 +129,19 @@ const {
 To customize the underlying DynamoDB's index name, you can optionally provide the `name()` modifier.
 
 ```ts
-const schema = a.schema({
-  Customer: a.model({
-    name: a.string(),
-    phoneNumber: a.phone(),
-    accountRepresentativeId: a.id().required(),
-  }).secondaryIndexes(index => [
-    index('accountRepresentativeId')
-    // highlight-next-line
-      .name("MyCustomIndexName"),
-  ])
-}).authorization([a.allow.owner()]);
+const schema = a
+  .schema({
+    Customer: a
+      .model({
+        name: a.string(),
+        phoneNumber: a.phone(),
+        accountRepresentativeId: a.id().required(),
+      })
+      .secondaryIndexes((index) => [
+        index("accountRepresentativeId")
+          // highlight-next-line
+          .name("MyCustomIndexName"),
+      ]),
+  })
+  .authorization([a.allow.owner()]);
```

Comment:
Can we move this authorization to the model level instead of the schema level to align with the rest of the code samples?

Reply:
Good catch! moved it up to the model level